### ISSUE TITLE
Bug: ANSI Escape Sequence present in WSL Bash Terminal (1.90.0-insider)

### ISSUE THREAD

**Issue by RDhar** [microsoft/vscode#211884]
Does this issue occur when all extensions are disabled?: Yes

Version: 1.90.0-insider (user setup)
Commit: 03dd3c3b93322b41fe108a3448308595ee7077a4
Date: 2024-05-02T05:50:23.338Z
Electron: 28.2.8
ElectronBuildId: 27744544
Chromium: 120.0.6099.291
Node.js: 18.18.2
V8: 12.0.267.19-electron.0
OS: Windows_NT x64 10.0.26100

Steps to Reproduce:

1. On WSL: Ubuntu, open default Bash terminal.
2. Observe as ANSI escape code sequences are present before the prompt (screenshot attached below).
    ```sh
    \x1b[01\x3b32mrdhar@pad\x1b[00m:\x1b[01\x3b34m~\x1b[00m$ rdhar@pad:~$ 
   ```
    ![image](https://github.com/microsoft/vscode/assets/19497993/5dd71853-ed45-496f-a843-94e41729dd09)

Expected Behaviour:

1. On WSL: Ubuntu, the terminal should not be preceded by ANSI escape code sequences.
2. The erroneous behavior is only present in WSL: Ubuntu, and absent from the regular VS Code or the [Terminal](https://github.com/microsoft/terminal) app itself.

**Comment by mitchelljqegan** [comment]
+1 Ubuntu 20.04

Version: 1.90.0-insider
Commit: 03dd3c3b93322b41fe108a3448308595ee7077a4
Date: 2024-05-02T05:49:28.932Z
Electron: 28.2.8
ElectronBuildId: 27744544
Chromium: 120.0.6099.291
Node.js: 18.18.2
V8: 12.0.267.19-electron.0
OS: Linux x64 5.15.0-100-generic

**Comment by TheGlitch1** [comment]
I don't have this issue with vscode terminal integration.

The ainsi escape sequence is displayed when I use bash inside phpstorm. Still looking for a fix.

**Comment by thealjey** [comment]
This exact same thing has also started to happen to me today with Git Bash on Windows.

**Comment by alexclarkmusa** [comment]
Happening for me right now on the Powershell terminal after the 1.90 update:

![image](https://github.com/microsoft/vscode/assets/61800093/c574cb5b-7055-4995-ba9c-5b23e5729e2a)

**Comment by zillemarco** [comment]
Same for me, happening after the auto-update to 1.90:

![image](https://github.com/microsoft/vscode/assets/9462123/a2628e5b-db68-4cdd-91dd-e2e21046aa23)

**Comment by alexclarkmusa** [comment]
Tried a Git Bash terminal and it works fine; so far it only seems to be Powershell.

I tried removing any custom settings relating to the terminal (e.g. terminal.integrated.gpuAcceleration) inside my Code\User\Settings.json file but to no avail.

Tried rebooting but it did not fix anything.

To be clear, this issue only exists in the VSCode integrated terminal, _not_ in the regular Windows Terminal for Powershell (or any other) instances.

Some additional info.
Calling $PSVersionTable from within VSCode gives me this:
![image](https://github.com/microsoft/vscode/assets/61800093/77fb8f9f-cb1a-413b-91d0-a831bd47ad82)

OS:
![image](https://github.com/microsoft/vscode/assets/61800093/2f7ab9ee-d09c-4625-8197-9098de16f27d)
